enterprise/server/configure.md

```diff
@@ -22,49 +22,49 @@ The Enhanced Media Embed server-side component require additional configuration,
 
 The `allowed-origins` element configures a list of **all** values that can be expected by the server-side components in a HTTP Origin header from your {{site.productname}} instances (see the [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) for more information on the HTTP Origin header). In short, you'll need to supply a list of all the URLs that your {{site.productname}} instances will be served from without the path information.
 
+> **Note:** When the services are matching against the `allowed-origins` list, the `scheme` (e.g. http) and the `port` (e.g. 800) are **ignored**. 
+
 This is best illustrated with some examples:
 
 If users load {{site.productname}} from the following URLs:
 
 - `http://server.example.com/editor.php`
 - `http://server.example.com/subpage/editor.php`
 
-Add `http://server.example.com` to the `allowed-origins` list.
+Add `server.example.com` to the `allowed-origins` list.
 
 If users load {{site.productname}} from the following URLs:
 
 - `https://server.example.com/editor.php`
 - `http://server.example.com/subpage/editor.php`
 
-Add `http://server.example.com` and `https://server.example.com` to the `allowed-origins` list.
+Add `server.example.com` to the `allowed-origins` list because the scheme is ignored.
```

Comment:
This wording is a little clumsy.

Reply:
I agree. Thoughts?

Reply:
```suggestion
Add `server.example.com` to the `allowed-origins` list (The scheme is ignored by the service).
```
?

Reply:
I thought about putting it in brackets as well

Reply:
This is pretty urgent. How much do you care?

Reply:
You know me well enough to know my answer to that question